styles/index.less

```
@@ -0,0 +1,60 @@
+@import "./common-styles/index";
+
+// TODO: asciinema is included in every build
+
+body {
+  & > header {
+    &:extend(.navigation-bar all);
+
+    margin-bottom: 0;
+  }
+
+  // TODO: why is this not possible?
+  //& > div.page-title {
+  //   &:extend(.page-title all);
+  //}
```

I think I see the problem.

The assumed structure for a page was:

```
body
  header
    ...
  main
    .page-title
    ...
```

This can be fixed by instead relying on

```less
header + div.page-title {
  margin-top: -1rem;     
}                        
```

Though this has to be fixed at the common styles repo to be more proper.


```suggestion
  & > header {
    &:extend(.navigation-bar all);
  }
```

---

https://github.com/NixOS/nixos-common-styles/pull/15